content/en/docs/reference/using-api/api-overview.md

```diff
@@ -26,17 +26,21 @@ platform is treated as an API object and has a corresponding entry in the
 
 ## API versioning
 
-The JSON and Protobuf serialization schemas follow the same guidelines for
-schema changes. The following descriptions cover both formats.
-
-The API versioning and software versioning are indirectly related.
-The [API and release versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
-describes the relationship between API versioning and software versioning.
+Kubernetes makes a strong commitment to maintain compatibility for official
+Kubernetes APIs once they reach general availability (GA), typically at API
+version `v1`. Additionally, Kubernetes keeps compatibility even for _beta_
+API versions wherever feasible:
```

Comment:
You want to duplicate the content from the concepts page?

Reply:
Indeed - I don't want to assume that the reader has read the other page.